DOCS.txt

```diff
@@ -161,9 +151,6 @@ I really do hate writing documents.
 	and see how long you can go.
 	- Giving high priorities to kernel_team's idle thread's is always a good thing
 	to do.
-	- Input server is a really unnecessary thing. We didn't have it in R3, so certainly
-	we don't need it in R4. So ya might as well kill it and get few extra CPU
-	cycles for some real work.
```

Comment:
I would keep this, maybe update it to refer to some other important server that was introduced in Haiku. package_daemon or launch_daemon maybe?
Or, if we want to make these docs really serious, we could remove that whole section.